ipatests/test_integration/test_cert.py

```diff
@@ -127,6 +128,91 @@ def test_ipa_getcert_san_aci(self):
         ipaddrs = ext.value.get_values_for_type(x509.IPAddress)
         assert ipaddrs == [ipaddress.ip_address(self.clients[0].ip)]
 
+    def test_getcert_list_profile(self):
+        """
+        Test that getcert list command displays the profile
+        for the cert
+        """
+        result = self.master.run_command(
+            ["getcert", "list", "-f", paths.HTTPD_CERT_FILE]
+        )
+        assert "profile: caIPAserviceCert" in result.stdout_text
+        result = self.master.run_command(
+            ["getcert", "list", "-n", "Server-Cert cert-pki-ca"]
+        )
+        assert "profile: caServerCert" in result.stdout_text
+
+    @pytest.fixture
+    def test_subca_certs(self):
+        """
+        Fixture to add subca, stop tracking request, 
+        followed by removing SUB CA along with
+        cert keys
+        """
+        sub_name = "CN=SUBCA"
+        tasks.kinit_admin(self.master)
+        self.master.run_command(
+            ["ipa", "ca-add", "mysubca", "--subject=" + sub_name]
+        )
+        self.master.run_command(
+            [
+                "ipa",
+                "caacl-add-ca",
+                "hosts_services_caIPAserviceCert",
+                "--cas=mysubca",
+            ]
+        )
+        yield
+        self.master.run_command(
+            ["getcert", "stop-tracking", "-i", "test-request"]
+        )
+        self.master.run_command(["ipa", "ca-disable", "mysubca"])
+        self.master.run_command(["ipa", "ca-del", "mysubca"])
+        self.master.run_command(
+            ["rm", "-frv", "/etc/pki/tls/private/test.key"]
```

Comment:
You don't need -r since you're removing a single file. Same below.

Reply:
removed the -r option